pennylane/transforms/condition.py

```diff
@@ -232,7 +247,71 @@ def qnode(a, x, y, z):
             >>> z = np.array(0.3, requires_grad=True)
             >>> qnode(par, x, y, z)
             tensor(-0.30922805, requires_grad=True)
+
+    In the compilation mode using the :func:`~.qjit` JIT compiler decorator,
+
+    .. code-block:: python
+
+        dev = qml.device("lightning.qubit", wires=1)
+
+        @qml.qjit
+        @qml.qnode(dev)
+        def circuit(x: float):
+            def ansatz_true():
+                qml.RX(x, wires=0)
+                qml.Hadamard(wires=0)
+
+            def ansatz_false():
+                qml.RY(x, wires=0)
+
+            qml.cond(x > 1.4, ansatz_true, ansatz_false)
+
+            return qml.expval(qml.PauliZ(0))
+
+    >>> circuit(1.4)
+    array(0.16996714)
+    >>> circuit(1.6)
+    array(0.)

```

Comment:
@maliasadi I would move this example to just before `.. details::`, because at the moment it is appearing after the 'Usage Details' section.